### DISCUSSION TITLE
How to add expiration headers to files in project hosted with vercel?

### DISCUSSION THREAD

**Discussion by qnxdev** [vercel/vercel#4590] [Help] [answered]
I have tried adding .htaccess inside build folder , which didn't work.

Thanks  in advance.

**Comment by qnxdev** [comment]
No response?

**Comment by styfle** [comment, marked as answer]
Hi @qnxdev 

You can create a `vercel.json` and add a `headers` property: https://vercel.com/docs/configuration#project/headers

Also take a look at our caching documentation as well: https://vercel.com/docs/v2/edge-network/caching
